packages/MarkdownEditor-Tests.package/MarkdownDelimiterEmphasisTest.class/instance/testDifferentDelimiterLengthsAttributeDelimiterLength.st

```diff
@@ -0,0 +1,11 @@
+tests - matching
+testDifferentDelimiterLengthsAttributeDelimiterLength
```

Comment:
What? xD 
What is this test case supposed to tell me that it is testing?

Reply:
Please suggest a better name We couldn't find a better one.